karpenter/datadog_checks/karpenter/metrics.py

```diff
@@ -126,6 +126,28 @@
     'workqueue_retries': 'workqueue_retries',
     'workqueue_unfinished_work_seconds': 'workqueue.unfinished.work_seconds',
     'workqueue_work_duration_seconds': 'workqueue.work.duration_seconds',
+    # new Karpenter V1 metric names
+    'karpenter_nodes_termination_duration_seconds': 'nodes.termination.time_seconds',
+    'karpenter_nodes_terminated_total': 'nodes.terminated',
+    'karpenter_nodes_leases_deleted_total': 'nodes.leases_deleted',
+    'karpenter_nodes_created_total': 'nodes.created',
```

Comment:
```suggestion
```

These we don't really need entries for. For counters we ignore the _total to begin with anyways. After stripping away the `_total` these look the same as they were before the change.